src/abi_transferable.rs

```diff
@@ -23,6 +23,14 @@ pub unsafe trait AbiTransferable {
         // correctly implement this trait
         unsafe { std::mem::transmute_copy(&abi) }
     }
+
+    fn slice_from_abi<'a>(abi: *const Self::Abi, len: usize) -> &'a [Self] {
```

Comment:
I think this function should be unsafe since it's up to the caller to ensure the pointer is valid for `'a`

Reply:
Agreed.